build.gradle

```diff
@@ -78,7 +78,7 @@ subprojects {
         testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:$junitJupiterVersion"
     }
 
-    tasks.withType(JavaCompile) {
+    tasks.withType(JavaCompile).configureEach {
```

Comment:
This was actually not suggested, took that suggestion from here though: https://github.com/tbroyer/gradle-errorprone-plugin#usage